documentation/smg/02_development_environment.md

```diff
@@ -213,5 +213,28 @@ At the time of writing, the only way I know to get these onto your laptop is to
 I use WinSCP. The user id and password are the same as for getting into ISIS Analysis (your STFC email and your chosen password),
 and the hostname can be found in the window title bar of the VM browser window -
 **NOT** the local name starting `host-192...` which you see in the individual terminal windows.
-Port is the default 22. An alternative tool is rsync, which I have not tried.
+Port is the default 22. An alternative tool is rsync, which I have not tried. 
+
+The Scratch location for this directory is not ideal, and to provide a more permanent home, 
+a zipped copy of this directory is now available at `\\isis.cclrc.ac.uk\Shares\PACE_Project_Tool_Source` (the SAN disk) 
+under the directory `PACE_Demo`. This disk is not mounted on IDaaS. 
+In the event that the `.../Scratch/Perring` location referred to above or its contents become unavailable, 
```

Comment:
Copying from a scratch location via Alex's server isn't relevant now the data is on the SAN. There is other documentation about accessing the windows build node a.k.a Alex's server.

Reply:
This isn't clear. What is the method of accessing the demo on IDaaS then if not by the route described?

Reply:
The problem with iDAaaS access to SAN storage is a temporary one. There is an open issue to address this problem #321.